recipe/conda_build_config.yaml

```
@@ -0,0 +1,2 @@
+MACOSX_SDK_VERSION:   # [x86_64]
+  - "11.0"            # [x86_64]
```

```suggestion
MACOSX_SDK_VERSION:   # [osx and x86_64]
  - "11.0"            # [osx and x86_64]
```

See https://conda-forge.org/docs/maintainer/knowledge_base/#requiring-newer-macos-sdks. otherwise you get a spurious `MACOSX_SDK_VERSION` value for all the non-osx systems.

---

Thanks a lot! Done in https://github.com/conda-forge/warp-lang-feedstock/pull/23/commits/6004c495aa4a24a6b18bdac9232f122b47142d1c